topcoffea/modules/corrections.py

```diff
@@ -454,14 +455,14 @@ def GetTriggerSF(year, events, lep0, lep1):
   ls=[]
   for i in [0,1,2]:
```

Comment:
What is this `i` doing exactly? Can you add a comment, or a descriptive variable name?

Reply:
`i` is just an index: i=0 for nominal SF, i=1 for SFdown and i=2 for SFup

Reply:
Could you change `i` to something like `syst` or `shift` and then put a comment for the nominal/up/down numbers?

Reply:
sure!

Reply:
Thanks!